src/containers/ProjectContentsContainer.jsx

```diff
@@ -50,16 +52,26 @@ class ProjectContentsContainer extends Component {
     this.setState({modalOpen: false});
   }
 
+  handleChange(event) {
+    this.setState({
+      translationText: event.target.value,
+    });
+  }
+
   handleClick(event) {
     this.setState({
+      field: event.target.textContent.split(':')[0],
+      fieldText: event.target.textContent.split(':')[1],
```

Comment:
You can look up the text from the original resource in state, once you know the field name.